src/app/resources/resources.component.ts

```diff
@@ -10,7 +10,10 @@ export class ResourcesComponent implements OnInit, AfterViewInit {
   resources = new MatTableDataSource();
   @ViewChild(MatPaginator) paginator: MatPaginator;
   displayedColumns = [ 'title', 'rating' ];
-  readonly dbName = 'resources';
+  readonly resourceDb = 'resources';
```

Comment:
I don't understand why readonly dbName = 'resources'; has been changed to readonly resourceDb = 'resources';

Reply:
@nirojdyola